content/en/technical-overview/architecture/_index.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
I suggest adding a CHT Android separate page where we could use elements from the [repo README](https://github.com/medic/cht-android) to explain the repo purpose and structure. 

Additionally, we might want to do the same for the `CHT Interoperability` product and add it to the architecture diagram.

Reply:
I agree about CHT-Android, but I wanted to keep this PR lightweight. I can add the page with text from the repo readme. 

Also agree about interop.